usr/local/www/guiconfig.inc

```diff
@@ -266,11 +266,15 @@ function do_input_validation($postdata, $reqdfields, $reqdfieldsn, &$input_error
 }
 
 function print_input_errors($input_errors) {
+    print_errors($input_errors, gettext('The following input errors were detected:'));
```

Comment:
Nothing. I was just following the technique used in the existing code. 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com 

> On Mar 28, 2015, at 11:19 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/guiconfig.inc <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27345716>:
> 
> > @@ -266,11 +266,15 @@ function do_input_validation($postdata, $reqdfields, $reqdfieldsn, &$input_error
> >  }
> >  
> >  function print_input_errors($input_errors) {
> > +    print_errors($input_errors, gettext('The following input errors were detected:'));
> What's wrong with writing the HTML directly?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27345716>.
>